src/components/UsedButton/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { useEffect } from 'react'
+import { useState } from 'react'
+import Button from '../Button'
+
+
+const UsedButton = ({
+                      firstButton,
+                      firstButtonText,
+                      secondButton,
+                      secondButtonText,
+                      firstArrowRight,
+                      FirstArrowLeft,
+                      secondArrowRight,
+                      secondArrowLeft,
+                    }) => {
+
+  const [ isDesktop, setDesktop ] = useState(window.innerWidth > 767)
+
+  const updateMedia = () => {
+    setDesktop(window.innerWidth > 767)
```

Comment:
so react responsive lib will ease your life and you don't have to write such code anymore